src/command_modules/azure-cli-interactive/azure/cli/command_modules/interactive/tests/test_query_injection.py

```diff
@@ -43,15 +43,51 @@ def test_null(self):
         self.shell.last.result = {}
         flag = self.shell.handle_jmespath_query(args)
         self.assertTrue(flag)
-        self.assertEqual('\n', self.stream.getvalue())
+        self.assertEqual(self.stream.getvalue(), '\n')
 
-    def test_just_query(self):
+
+    def test_print_last_command(self):
+        # tests getting last command result
+        args = ['??']
+        self.shell.last.result = {'x': 'result'}
+        flag = self.shell.handle_jmespath_query(args)
+        self.assertTrue(flag)
+        print(repr(self.stream.getvalue()))
```

Comment:
You don't need to print in tests. If the following assertion fails, the test harness should be in charge of print the values.

Reply:
forgot to remove them, thanks